src/main.rs

```diff
@@ -268,7 +273,8 @@ describe! main {
             assert_eq!(args.flag_local_name, None);
             assert_eq!(args.flag_port, 3000);
             assert_eq!(args.flag_wsport, 4000);
-            assert_eq!(args.flag_register, "http://localhost:4242/register");
```

Comment:
Right, I just ran into this as well, current master branch tries to post to http://localhost:4242/register/register :)